docs/content/en/docs/concepts/workloads/_index.md

```diff
@@ -7,21 +7,34 @@ weight: 10
 hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-A Workload contains information about which tasks should be performed during the `preDeployment` as well as
-the `postDeployment`
-phase of a deployment.
-In its state it keeps track of the currently active `Workload Instances`, which are responsible
-for doing those checks for
-a particular instance of a Deployment/StatefulSet/ReplicaSet (e.g. a Deployment of a certain version).
+A `KeptnWorkload`resource contains information about
+which tasks should be performed during the `preDeployment`
+or `postDeployment` phase of a deployment.
+In its state,
+it keeps track of the currently active `Workload Instances`,
+which are responsible for doing those checks
+for a particular instance of a Deployment/StatefulSet/ReplicaSet
+(e.g. a Deployment of a certain version).
 
-### Keptn Workload Instance
+## KeptnWorkload
 
-A Workload Instance is responsible for executing the pre- and post deployment checks of a workload.
-In its state, it
-keeps track of the current status of all checks, as well as the overall state of
-the Pre Deployment phase, which can be used by the scheduler to tell that a pod can be allowed to be placed on a node.
-Workload Instances have a reference to the respective Deployment/StatefulSet/ReplicaSet, to check if it has reached the
-desired state.
-If it detects that the referenced object has reached
-its desired state (e.g. all pods of a deployment are up and running), it will be able to tell that
-a `PostDeploymentCheck` can be triggered.
+A `KeptnWorkload` resource augments a Kubernetes
+[Workload](https://kubernetes.io/docs/concepts/workloads/)
+with the ability to handle extra phases.
+KLT generates the `KeptnWorkload` resource
+from metadata information;
+it is not necessary to manually create a YAML file that defines it.
+
+A `KeptnWorkload` instance is responsible for executing
+the pre- and post deployment checks of a workload.
+In its state, it keeps track of the current status of all checks,
+as well as the overall state of the Pre Deployment phase,
+which the scheduler can use to determine
```

Comment:
I do not agree with this sentence: yes the scheduler determines if a node meets the requirements to bind a pod, but the pre-deployment phase can be used for many kind of checks, while now it sound like this is only to check on nodes? I would rephrase it without mentioning that it uses it to decide node binding:

```suggestion
which the scheduler can use to determine whether the deployment should proceed and whether the pod should be bound to a node or stay pending.
```